--- src/ssl.orig.c	2021-09-20 09:16:59.458904925 +0100
+++ src/ssl.c	2021-09-20 09:22:17.952243782 +0100
@@ -127,7 +127,7 @@
 		return LIBIRC_ERR_SSL_INIT_FAILED;
 
 	// Disable SSLv2 as it is unsecure
-	if ( (SSL_CTX_set_options( ssl_context, SSL_OP_NO_SSLv2) & SSL_OP_NO_SSLv2) == 0 )
+	if ( (SSL_CTX_set_options( ssl_context, SSL_OP_NO_SSLv2) & SSL_OP_NO_SSLv2) != SSL_OP_NO_SSLv2 )
 		return LIBIRC_ERR_SSL_INIT_FAILED;
 
 	// Enable only strong ciphers
